src/concepts/components/README.md

```diff
@@ -4,171 +4,48 @@ description: Components and their lifecycle hooks
 
 # Components
 
-## What are Components?
+## What are Components
 
 Components are the building blocks of Yew. They manage their own state and can render themselves to the DOM. Components are created by implementing the `Component` trait which describes the lifecycle of a component.
 
-## Lifecycle
-
-{% hint style="info" %}
-`Contribute to our docs:` [Add a diagram of the component lifecycle](https://github.com/yewstack/docs/issues/22)
-{% endhint %}
-
-## Lifecycle Methods
-
-### Create
-
-When a component is created, it receives properties from its parent component as well as a `ComponentLink`. The properties can be used to initialize the component's state and the "link" can be used to register callbacks or send messages to the component.
-
-It is common to store the props and the link in your component struct, like so:
+## Simple Component
 
 ```rust
-pub struct MyComponent {
-    props: Props,
-    link: ComponentLink<Self>,
-}
+use yew::prelude::*;
+pub struct SimpleComponent {}
 
-impl Component for MyComponent {
-    type Properties = Props;
-    // ...
+impl Component for SimpleComponent {
```

Comment:
I'd like to see the simple component do some simple things like accept a property, handle a message, and re-render itself. If I were new to Yew, I'd be wondering what `_link` would be used for and why this component returns true in `update()` and `change()`. I think there should be any functionality that a new dev could look at a simple component and get a sense for how it works in practice

Reply:
I will think about it ... 
My issues that may goes against the reading flow extending more and more the example.
I will think ;-)

Reply:
haha yeah it's a difficult balance! Looking forward to what you come up with :)